scripts/build-browser.js

```diff
@@ -7,6 +7,29 @@ build({
   bundle: true,
   sourcemap: true,
   format: 'iife',
+  banner: {
+    js: `"use strict";
+(function universalModuleDefinition(root, factory) {
+  if(typeof exports === 'object' && typeof module === 'object')
+    module.exports = factory();
+  else if(typeof define === 'function' && define.amd)
+    define([], factory);
+  else if(typeof exports === 'object')
+    exports["SolidityParser"] = factory();
+  else
+    root["SolidityParser"] = factory();
+})(
+typeof globalThis !== 'undefined' ? globalThis
+: typeof global !== 'undefined' ? global
+: typeof self !== 'undefined' ? self
+: this || {}
+, () => {`,
+  },
   globalName: 'SolidityParser',
-  outfile: path.resolve(__dirname, '../dist/index.iife.js'),
+  outfile: path.resolve(__dirname, '../dist/index.umd.js'),
+  footer: {
+    js: `
+  return SolidityParser;
+})`,
```

Comment:
Maybe let's add a semicolon for consistency here, and to prevent potential problems if someone is concatenating this with other files? (Extremely unlikely, but it doesn't hurt)

```suggestion
});`,
```